packages/dds/sequence-sequential-id/src/SharedStringWithSequentialIdFactory.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as MergeTree from "@fluidframework/merge-tree";
+import { IChannelAttributes, IChannelServices,
+    IFluidDataStoreRuntime, IChannelFactory } from "@fluidframework/datastore-definitions";
+import { SharedString } from "@fluidframework/sequence";
+import { pkgVersion } from "./packageVersion";
+import { sharedStringWithSequentialIdMixin } from "./sharedStringWithSequentialId";
+
+export class SharedStringWithSequentialIdFactory implements IChannelFactory {
+    // TODO rename back to https://graph.microsoft.com/types/mergeTree/string once paparazzi is able to dynamically
+    // load code
+    public static Type = "https://graph.microsoft.com/types/mergeTree";
```

Comment:
It has to have it's own type, it can't use same type. It will be breaking change - we will need to
1. Deploy bits with this factory first (but no other changes)
2. after a week or so, for new document only start using this new factory, but leave old docs using old factory.

Reply:
Renamed type